nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -151,11 +151,13 @@ in
       };
 
       daemonIONiceLevel = mkOption {
-        type = types.int;
+        type = types.either types.int (types.enum ["idle"]);
```

Comment:
What if we had "idle", "low" (best-effort, 7), "inherit" (whatever the OS determines), and "high" (best-effort, 0). Does anybody care about granular best-effort levels beyond lowest and highest?